tests/Controller/EnMarche/AdherentControllerTest.php

```diff
@@ -104,10 +104,10 @@ public function testProfileActionIsAccessibleForAdherent($profilePage, $title)
 
     public function provideProfilePage()
     {
-        yield ['/espace-adherent/mon-profil', 'Informations personnelles'];
-        yield ['/espace-adherent/mon-profil/centres-d-interet', 'Centres d\'intérêt'];
-        yield ['/espace-adherent/mon-profil/changer-mot-de-passe', 'Mot de passe'];
-        yield ['/espace-adherent/mon-profil/preferences-des-emails', 'Préférences des e-mails'];
+        yield ['/espace-adherent/mon-compte', 'Informations personnelles'];
+        yield ['/espace-adherent/mon-profil', 'Profil En Marche'];
```

Comment:
Yes.
I have already removed this line, but not push commit. It also corrige tests.